km_api/know_me/tests/models/test_km_user_accessor_model.py

```diff
@@ -20,4 +20,5 @@ def test_string_conversion(km_user_accessor_factory):
     """
     accessor = km_user_accessor_factory()
 
-    assert str(accessor) == "Accessor for " + accessor.km_user.name
+    expected = 'Accessor for {user}'.format(user=accessor.km_user.name)
```

Comment:
They currently return the same thing, but if the `__str__` method of the `KMUser` model were to change, what would you want the `KMUserAccessor` model's `__str__` method to return?